great_expectations/data_context/store/store.py

```diff
@@ -1,38 +1,35 @@
-import logging
-logger = logging.getLogger(__name__)
-
-import json
-import os
-import io
-import six
-
-import pandas as pd
-
-from ..util import safe_mmkdir
-
+from ..types import (
+    NameSpaceDotDict,
+)
 from .types import (
     InMemoryStoreConfig,
     FilesystemStoreConfig,
 )
-from ..types import (
-    NameSpaceDotDict,
-)
+from ..util import safe_mmkdir
+import pandas as pd
+import six
+import io
+import os
+import json
+import logging
+logger = logging.getLogger(__name__)
+
 
 class ContextAwareStore(object):
     def __init__(
         self,
         data_context,
         config,
     ):
-        #FIXME: Eek. This causes circular imports. What to do?
-        #TODO: remove the dependency. Stores should be based on namespaceIdentifier objects, but not Context itself.
+        # FIXME: Eek. This causes circular imports. What to do?
```

Comment:
This is the reason for NamespaceAwareStore, right?

Reply:
Yes, that's right.